static/javascripts/kp_extraction_emb.js

```diff
@@ -1,37 +1,103 @@
 function submit(input_text, number_keyphrases) {
-	$("#last_group").append('<i class="fa fa-spinner fa-spin" id="spinner"></i>');
-	console.log("url input: ", input_text, 'number keyphrases', number_keyphrases);
-	$('#input_text').val('');
-	url = "/kp/emb";
-	data = {'inp_url' : input_text,
-	  'nbkp' : number_keyphrases
-	  };
-	$.ajax({
-	  type: "POST",
-	  url: url,
-	  contentType: 'application/json',
-	  data: JSON.stringify(data, null, '\t'),
-	  success: function(data){
-	  	console.log(data);
-	  	$('#kpvizu_div').html(data);
-	  	$("#spinner").remove();
-	  }
-	});
+    $("#last_group").append('<i class="fa fa-spinner fa-spin" id="spinner"></i>');
+    $('#input_text').val(input_text);
+
+    url = "http://52.30.254.33:1277/api/kp";
+    data = {
+        'inp_url': input_text,
+        'nbkp': number_keyphrases
+    };
+    $.ajax({
+        url: "http://52.30.254.33:1277/api/kp",
+        type: "POST",
+        contentType: 'application/json',
+        dataType: "json",
+        data: JSON.stringify(data),
+        success: function (data) {
+            //Process the JSON
+            $("#spinner").remove();
+            var itemList = document.getElementById("list-group");
+
+            //Loop trough the JSON
+            data
+                .api_result
+                .forEach(function (element) {
+                    element.relevance *= 100;
+                    var probability = element.relevance;
+                    itemList.insertAdjacentHTML('beforeend', "<li data-time='" + probability + "' class='list-group-item' style='background : linear-gradient(90deg, #c6e3fd " + probability + "%, #FFFFFF 0%); text-shadow: none'>  <b>" + element.label + "</b> </li>");
+                    if (element.aliases.length > 0) {
+                        for (var i = 0; i < element.aliases.length; i++) {
+                            probability -= 0.000001;
```

Comment:
Why do you do that?

Reply:
you mean line 30 I think?

Reply:
Yes, why do you do this substraction ?

Reply:
Because without this sometimes it  happen that it's not grouped right. So this substraction is for grouping right. Maybe is another function better idk